android/KMAPro/kMAPro/src/main/java/com/keyman/android/SystemKeyboard.java

```diff
@@ -165,6 +165,7 @@ public void onStartInput(EditorInfo attribute, boolean restarting) {
   @Override
   public void onStartInputView(EditorInfo attribute, boolean restarting) {
     super.onStartInputView(attribute, restarting);
+    setInputView(onCreateInputView());
```

Comment:
Is there a fix for the in-app keyboard because I had issues while rotating that one too?

Reply:
I haven't seen the rotation issues for the in-app keyboard? We'll need to investigate that separately I think. Can you document a repro?

Reply:
@MakaraSok attached some screenshots while testing https://github.com/keymanapp/keyman/pull/5201#issuecomment-853650978

Reply:
We probably also want this on SystemKeyboard for KMSample2 and FV app